tests/e2e/pageObjects/slow-log-page.ts

```diff
@@ -0,0 +1,103 @@
+    slowLogConfigureMilliSecondsUnit = Selector('[data-test-subj=unit-milli-second]');
+    slowLogConfigureMicroSecondsUnit = Selector('[data-test-subj=unit-micro-second]');
+    slowLogSaveConfigureButton = Selector('[data-testid=slowlog-config-save-btn]');
+    slowLogCancelConfigureButton = Selector('[data-testid=slowlog-config-cancel-btn]');
+    slowLogDefaultConfigureButton = Selector('[data-testid=slowlog-config-default-btn]');
+    slowLogRefreshButton = Selector('[data-testid=refresh-slowlog-btn]');
+    slowLogClearButton = Selector('[data-testid=clear-btn]');
+    slowLogConfirmClearButton = Selector('[data-testid=reset-confirm-btn]');
+    //INPUTS
+    slowLogSlowerThanConfig = Selector('[data-testid=slower-than-input]');
+    slowLogMaxLengthConfig = Selector('[data-testid=max-len-input]');
+    //TEXT ELEMENTS
+    slowLogTimestampValue = Selector('[data-testid=timestamp-value]');
+    slowLogDurationValue = Selector(this.cssSelectorDurationValue);
+    slowLogCommandValue = Selector('[data-testid=command-value]');
+    slowLogEmptyResult = Selector('[data-testid=empty-slow-log]');
+    slowLogCommandStatistics = Selector('[data-testid=entries-from-timestamp]');
+    // Table
+    slowLogTable = Selector('[data-testid=slowlog-table]');
+
+    /**
+     * Set value for slowlog-log-slower-than parameter
+     * @param slowerThan Value for slowlog-log-slower-than property
+     * @param unit Value for unit property
+     */
+    async changeSlowerThanParameter(slowerThan: number, unit?: Selector): Promise<void> {
+        await t.click(this.slowLogConfigureButton);
+        await t.typeText(this.slowLogSlowerThanConfig, slowerThan.toString(), { replace: true });
+        if (unit !== undefined) {
+            // await t.debug();
+            await t.click(this.slowLogConfigureUnitButton);
+            await t.click(unit);
+        }
+        await t.click(this.slowLogSaveConfigureButton);
+    }
+
+    /**
+     * Set value for slowlog-max-len parameter
+     * @param maxLength Value for slowlog-max-len property
+     */
+    async changeMaxLengthParameter(maxLength: number): Promise<void> {
+        await t.click(this.slowLogConfigureButton);
+        await t.typeText(this.slowLogMaxLengthConfig, maxLength.toString(), { replace: true });
+        await t.click(this.slowLogSaveConfigureButton);
+    }
+
+    /**
+     * Compare command displayed in the table with specified slowlog-log-slower-than parameter
+     * @param command command for comparing
+     * @param slowerThan time to compare
+     */
+    async compareCommandExecutionWithSlowerThan(command: string, slowerThan: number): Promise<void> {
+        // Get duration of specified command
+        const duration = await this.slowLogCommandValue.withExactText(command).parent(3).find(this.cssSelectorDurationValue).textContent;
+        await t.expect(parseInt(duration)).gte(slowerThan, 'Displayed command time execution is more than specified');
```

Comment:
We generally do not use assertions on POM.
These functions could be on the common actions directory. (Relevant to this function, and those below).

Reply:
Changed all the functions